packages/builder/src/registry.ts

```diff
@@ -480,7 +480,36 @@ export class OnChainRegistry extends CannonRegistry {
       ],
     });
 
-    return (url as string) || null;
+    const response = await axios.post(
```

Comment:
This call could be moved to a helper like `contractFunctionCall` so this method does not get super long.